src/corehost/common/pal.windows.cpp

```diff
@@ -199,14 +199,24 @@ bool pal::get_default_servicing_directory(string_t* recv)
 
 bool pal::get_global_dotnet_dirs(std::vector<pal::string_t>* dirs)
 {
-    pal::string_t dir;
-    if (!get_default_installation_dir(&dir))
+    pal::string_t default_dir;
+    pal::string_t custom_dir;
+    bool gdir_found = false;
```

Comment:
What is `gdir`? I assume it means "global dir" per the method name. In that case please don't use abbreviations which are not common. Anyway, I think just `dir_found` would work well here.